### DISCUSSION TITLE
[#991932] List of settings to change for getting achievements in editor mode can be empty

### DISCUSSION THREAD

**Discussion by TORD31** [Mojang/minecraft-editor#67] [Bugs]
In editor mode, you can't earn any achievements (which makes sense). When you create a new project, you have the tipical hint that says: "You can't earn Achievements. Read more". But when you than click on "Read more" you get an empty list of settings you need to change to earn achievements (unless you change settings which deactivate achievements in normal minecraft. Then these are listed)

Repro steps
1. Open the editor mode
2. Click on "Create New Project"
3. Click on "Read more" in the achievements hint

Observed result vs. what you actually expected to happen
The list of settings to change before you can earn achievments again is empty. I don't expect to see any list of settings there because you can't get achievments in editor mode (it doesn't matter which settings you made).
![achievement list bug](https://user-images.githubusercontent.com/127531352/224368246-bfb66db7-ed19-4c32-b913-9207cd817a0c.png)

**Comment by minecraftchloe** [comment]
Great find! We need to close that loophole and confusion - no achievements in Editor Mode :)

**Comment by rpoerschke** [comment]
Issue addressed in 0.3.3.  Thanks!
